core/api-changes.json

```diff
@@ -75,6 +75,15 @@
           "classSimpleName": "ReactiveInterceptor",
           "elementKind": "interface",
           "justification": "MULE-18177: This should have neven been made privileged"
+        },
+        {
+        "code": "java.method.addedToInterface",
```

Comment:
Is this necessary after adding the default implementation?

Reply:
let me check